src/storage/src/hummock/mod.rs

```diff
@@ -373,7 +374,9 @@ pub async fn get_from_sstable_info(
     // Iterator gets us the key, we tell if it's the key we want
     // or key next to it.
     let value = if iter.key().user_key == *ukey {
-        if delete_epoch
+        if iter.key().epoch <= min_epoch {
+            None
+        } else if delete_epoch
```

Comment:
`e2e_test` indicated we lost a delete, I think something wrong here?

Reply:
I mispassed the prefix_hint for point get